web/themes/custom/pineapple/components/_accordion-item.twig

```diff
@@ -0,0 +1,7 @@
+<button class="accordion-title">{{ accordion_title }}
+  <span class="accordion-icon plus" >+</span>
+  <span class="accordion-icon minus" >-</span>
+</button>
+<div class="accordion-content" data-tab-content>
```

Comment:
What's the purpose of data-tab-content? i don't see it being used anywhere in the codebase

Reply:
I'm very sorry that came from a code snippet I had in my tech plan and I really thought I deleted it.